index.html

```diff
@@ -733,6 +733,12 @@ <h3>BitstringStatusListEntry</h3>
 credential.
         </p>
 
+        <p class="note">
+It is important to note that the statusListIndex is the only link between
+the [=Verifiable credential=] and its status in the list. The
+```credentialSubject.id``` is not used when checking status. 
```

Comment:
Formatting fixes (I think) and a slight tweak to make `credentialSubject.id` just one example -- and to change the "when" conjunction so we aren't suggesting that some business rule can't consider other properties "during" a status check, but rather, those properties aren't used *to* check status, or more specifically for the purpose of VC status list linkage. I believe we are trying to say these other properties aren't the relevant "links" to status here, and this is the best language I could quickly come up with.

```suggestion
the [=verifiable credential=] and its status in the list. Other
properties such as `credentialSubject.id` are not used for
this purpose.
```

Reply:
Added manually.